ProfileManager.js

```diff
@@ -14,17 +14,30 @@ import {
 import Collection from './Collection.js';
 import {EdvClient, EdvDocument} from 'edv-client';
 import EdvClientCache from './EdvClientCache.js';
+import LRU from 'lru-cache';
 import keyResolver from './keyResolver.js';
 import utils from './utils.js';
 import assert from './assert.js';
 
 const JWE_ALG = 'ECDH-ES+A256KW';
+const EDVS = {
```

Comment:
These aren't `EDVS` ... can we get a better name here? What are these... ZCAP reference IDs?

`ZCAP_REFERENCE_IDS` ?

Reply:
Renamed.